# Generated from: /home/atty/Prog/__IGN__/2015_LI3DS/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogMag.h
# SbgLogMag
uint32 timeStamp
uint16 status
float32[3] magnetometers
float32[3] accelerometers